src/main/scala/autoversion/AutoVersionPlugin.scala

```diff
@@ -30,7 +30,7 @@ object AutoVersionPlugin extends AutoPlugin {
     unreleasedCommits := listUnreleasedCommits.value,
     suggestedBump := suggestBump.value,
     releaseVersion := { ver =>
-      Version(ver).map(v => v.bump(suggestedBump.value).withoutQualifier.string).getOrElse(versionFormatError)
+      Version(ver).map(v => v.bump(suggestedBump.value).withoutQualifier.string).getOrElse(versionFormatError(ver))
```

Comment:
AFAICT this is the only change in `sbt-release` between 1.0.6 and 1.0.15 that needs addressing.

The test fails on 1.0.6 as well, so this isn't critical for the PR.